extensions/panache/mongodb-panache/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -300,6 +301,31 @@ void buildMutiny(CombinedIndexBuildItem index,
         }
     }
 
+    @BuildStep
+    ValidationPhaseBuildItem.ValidationErrorBuildItem validate(ValidationPhaseBuildItem validationPhase,
+            CombinedIndexBuildItem index) throws BuildException {
```

Comment:
```suggestion
            CombinedIndexBuildItem index)  {
```

No exception is thrown

Reply:
It can be thrown by `io.quarkus.panache.common.deployment.JandexUtil.isSubclassOf` that delcares it as a checked exception.